examples/quantum_die.py

```diff
@@ -1,79 +1,45 @@
-#!/usr/bin/env python
```

Comment:
Please do not delete the header of this file, including the license and copyright notice.

Do note that your contributions are recorded as a part of the commit history.

Reply:
Please also run `pep8` on this file and make the necessary fixes to make it PEP8 conforming.